plugins/SitesManager/vue/src/SiteFields/SiteFields.vue

```diff
@@ -452,6 +454,20 @@ export default defineComponent({
         this.$emit('delete', this.theSite);
       });
     },
+    getMessagesToWarnOnSiteRemoval() {
+      AjaxHelper.post({
+        idSite: this.theSite.idsite,
+        module: 'API',
+        format: 'json',
+        method: 'SitesManager.getMessagesToWarnOnSiteRemoval',
+      }).then((response) => {
+        this.deleteSiteExplanation = translate('SitesManager_DeleteSiteExplanation');
+        if (response.value) {
+          this.deleteSiteExplanation = response.value;
```

Comment:
according to the suggested changes above this would then need to be:

```suggestion
          this.deleteSiteExplanation += response.value.join('<br>');
```